docs/jplspec/jplspec.rst

```diff
@@ -235,7 +234,7 @@ response length:
 .. doctest-remote-data::
 
    >>> print(result.meta['comments'])
-   ['', '', '', '', '', 'form is currently limited to 2000 lines. Please limit your search.']
+   ['', '', '', '', '', '', '', '', '', '', '', 'form is currently limilted to 2000 lines. Please limit your search.']
```

Comment:
```suggestion
   ['', '', '', '', '', '', '', '', '', '', '', 'form is currently limited to 2000 lines. Please limit your search.']
```

Reply:
Don't fix the too, this is what comes from the server 😅

Reply:
OH! Wow.  🤦